docs/vsphere_ci/README.md

```diff
@@ -99,6 +100,16 @@ To enable detailed logging:
   PACKER_LOG=1 packer build build.json
 ```
 
+### What to do during the Packer build 
+
+During the golden image creation, it is highly recommended to establish access to the virtual machine by launching a
+Web Console through the vSphere Client. This can be done after the Packer build has powered on the VM (while it is
+<i>Waiting for IP...</i>). 
+
+If the build halts and prompts for a product key during the Windows OS setup, manual intervention will be required.
+When accessing the golden image via Web Console, send a `Ctrl+Alt+Del` then tab over to `I don't have a product key`,
+and hit enter on the keyboard. This should start the OS setup as intended.
```

Comment:
Is this required even after [providing](https://github.com/openshift/windows-machine-config-operator/pull/892/commits/848749c1def7ab9435837f9abd25134dd1ebee00#diff-f95d5d0ee98dedf392fb54c0fb93f7e3df3c9699b01d08ca8fd00d30ccba8244R66) the `ProductKey`?

Reply:
It was for both Sebastian and I.

Reply:
Maybe a user will figure out whats going on here and open an issue helping us out :)

Reply:
I follow, manual intervention is required. Why adding the `ProductKey` section in [autounattend.xml](https://github.com/openshift/windows-machine-config-operator/pull/892/commits/9826af4103d03e40a0d2d8466854c3e341a0d700#diff-f95d5d0ee98dedf392fb54c0fb93f7e3df3c9699b01d08ca8fd00d30ccba8244).

Reply:
I suggest removing it, in case the ProductKey section is not required in [autounattend.xml](https://github.com/openshift/windows-machine-config-operator/pull/892/commits/9826af4103d03e40a0d2d8466854c3e341a0d700#diff-f95d5d0ee98dedf392fb54c0fb93f7e3df3c9699b01d08ca8fd00d30ccba8244).

Reply:
How do we know removing it will not break golden image creation though?

Reply:
Testing the golden image creation without the ProductKey section in [autounattend.xml](https://github.com/openshift/windows-machine-config-operator/pull/892/commits/9826af4103d03e40a0d2d8466854c3e341a0d700#diff-f95d5d0ee98dedf392fb54c0fb93f7e3df3c9699b01d08ca8fd00d30ccba8244).

Reply:
When creating template without ProductKey in autounattend.xml, golden image VM fails to complete Microsoft Server Operating System Setup with message: `Windows cannot read the <ProductKey> setting from the unattend answer file`